submissions/adebayoObafemi/database/query.sql

```diff
@@ -0,0 +1,36 @@
+use transportation_company;
+
+/* Question 1: What is the total number of passengers who survived? 
+	Answer: 342*/
+select count(survived) from accidents where survived = 1;
+
+
+/* Question 2: What is the total number of passengers who did not survive? 
+	Answer: 549*/
+select count(survived) from accidents where survived = 0;
+
+
+/* Question 3: Get the name and sex of passengers under the age of 27 
+that embarked at Queenstown and Cherbourg? */
+select id, full_name, sex, age
+from passengers
+left join trips
+on passengers.id = trips.passenger_id
+where passengers.age < 27 and trips.point_of_embarkation = 'Q' or 'C';
+
+
+/* Question 4: How many of the passengers that embarked at Southampton survived?
+	Answer: 218*/
+select count(survived) from accidents 
+right join trips
+on accidents.passenger_id = trips.passenger_id
+where accidents.survived = 1 and trips.point_of_embarkation = 'S';
+
+
+/* Question 5: Get the id, name and the total number of passengers
+ who paid a fare greater than $100 and above 
+ the age of 35 had siblings or spouses on board? */
+select id, full_name, count(full_name) from passengers
+left join trips
+on passengers.id = trips.passenger_id
+where passengers.age > 35 and trips.trip_fare > 100 and trips.no_of_siblings_or_spouses_aboard > 0;
```

Comment:
Get the list of id, names and the total number of passengers  who paid a fare greater than $100 and above 
 the age of 35 had siblings or spouses on board as instructed  above,

Reply:
Hello @bayooba I would like to see your solutions to this last question. The total number and the list of passengers w

Reply:
Alright. I will do that and revert.

Reply:
I have included the result for the count in the SQL file. Attached is the screenshot of the solution to the first part for the question.
![Screenshot (5)](https://user-images.githubusercontent.com/67081020/202849736-d9dc49c4-c045-48fd-af6f-72755520d0a6.png)

Reply:
The updates have been included in the code already. I have attached another screenshot that captures the full list of passengers.

![Screenshot (7)](https://user-images.githubusercontent.com/67081020/202864443-10ef8a18-6468-42f8-b9ad-a12b6daea98b.png)